lang/cs/Org.Apache.REEF.Client.Tests/WindowsHadoopEmulatorYarnClientTests.cs

```diff
@@ -91,19 +92,74 @@ public void TestGetClusterMetrics()
             Assert.IsTrue(clusterMetrics.ActiveNodes > 0);
         }
 
-        //// TODO: [REEF-757] Once submit API is in place, submit an app then get the details
-        ////[TestMethod]
-        ////[TestCategory("Functional")]
-        ////public void TestGetApplication()
-        ////{
-        ////    const string applicationName = @"application_1440795762187_0001";
+        [TestMethod]
+        [TestCategory("Functional")]
+        public void TestApplicationSubmissionAndQuery()
```

Comment:
If these tests are all testing ``async`` functions, you should probably consider changing these tests to have the ``async`` signature instead.